lib/data/menu_items.dart

```diff
@@ -20,16 +20,5 @@ class MenuItemRepository {
           itemColor: Color.fromARGB(255, 67, 240, 177),
           routeName: '/customlistviewfeedAndWidget',
         ),
-        MenuItemModel(
```

Comment:
@DanielFrTB - Why were these pages removed?

Reply:
I am not sure, why did you remove it @moshe-l-taboola  ?

Reply:
Regarding custom_scroll_view_page_webview_and_feed @moshe-l-taboola  decided to remove since we dont have active publishers that are using it.